### PR TITLE
Problem solved on 10/8

### PR DESCRIPTION
**Pull request by yuya-minamide**
There are problems on LeetCode which were solved on 10/8.

### CODE REVIEW

File: 1832.check-if-the-sentence-is-pangram.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        if (sentence.includes(alphabet[i])) {
+            counter++;
+        }
+    }
+    
+    if (counter >= 26) {
+        return true;
+    } else {
+        return false;
+    }
```
Comment (ryonryon) [inline 1832.check-if-the-sentence-is-pangram.js:30]:
```suggestion
    return counter >= 26;
```

File: 1832.check-if-the-sentence-is-pangram.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * @param {string} sentence
+ * @return {boolean}
+ */
+var checkIfPangram = function(sentence) {
+    
+    const alphabet = "abcdefghijklmnopqrstuvwxyz";
+    
+    let counter = 0;
+    
+    if (sentence.length < 26) {
```
Comment (ryonryon) [inline 1832.check-if-the-sentence-is-pangram.js:16]: where does 26 come from? Is it easy to understand for readers?

File: 1832.check-if-the-sentence-is-pangram.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * https://leetcode.com/problems/check-if-the-sentence-is-pangram/
+ */
+
+/**
+ * @param {string} sentence
+ * @return {boolean}
+ */
+var checkIfPangram = function(sentence) {
+    
+    const alphabet = "abcdefghijklmnopqrstuvwxyz";
```
Comment (ryonryon) [inline 1832.check-if-the-sentence-is-pangram.js:12]: Is there programmatic way to get this?

**Review by ryonryon — APPROVED** [review]

**Review by HidemichiShimura — APPROVED** [review]

File: 1967.number-of-strings-that-appear-as-substrings-in-word.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * @param {string[]} patterns
+ * @param {string} word
+ * @return {number}
+ */
+var numOfStrings = function(patterns, word) {
+    
+    let counter = 0;
+    
+    for (let i = 0; i < patterns.length; i++) {
+        if(word.includes(patterns[i]) === true) {
```
Comment (HidemichiShimura) [inline 1967.number-of-strings-that-appear-as-substrings-in-word.js:16]: You can remove the last part of the condition like this.
`if(word.includes(patterns[i])) {`

**Review by HidemichiShimura — APPROVED** [review]

File: 1832.check-if-the-sentence-is-pangram.js (hunk 2 above)
Comment (yuya-minamide) [reply 1832.check-if-the-sentence-is-pangram.js]: I corrected this 26 to alphabet.length.

File: 1832.check-if-the-sentence-is-pangram.js (hunk 3 above)
Comment (yuya-minamide) [reply 1832.check-if-the-sentence-is-pangram.js:12]: I could've used map() at this problem.

File: 1832.check-if-the-sentence-is-pangram.js (hunk 1 above)
Comment (yuya-minamide) [reply 1832.check-if-the-sentence-is-pangram.js]: I corrected this code.

File: 1967.number-of-strings-that-appear-as-substrings-in-word.js (hunk 4 above)
Comment (yuya-minamide) [reply 1967.number-of-strings-that-appear-as-substrings-in-word.js]: I corrected this code.

**Pull request merged**